src/internal/storage/option.go

```diff
@@ -0,0 +1,44 @@
+package storage
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/pachyderm/pachyderm/v2/src/internal/obj"
+	"github.com/pachyderm/pachyderm/v2/src/internal/serviceenv"
+	"github.com/pachyderm/pachyderm/v2/src/internal/storage/chunk"
+	"github.com/pachyderm/pachyderm/v2/src/internal/storage/fileset"
+	"github.com/pachyderm/pachyderm/v2/src/internal/uuid"
+)
+
+// FilesetOptions returns the fileset storage options for the config.
+func FilesetOptions(conf *serviceenv.Configuration) []fileset.StorageOption {
```

Comment:
Is there a reason why these needed to move? Like a dependency cycle?

I'd prefer to keep the storage directory empty.  We can call these `MakeOptions` or `OptionsFromConfig` or whatever name you want.

Reply:
I was fighting an import cycle from trying to allow `CreateFilesetCallback(context.Context, func(*fileset.UnorderedWriter) error)` to exist in the pfs api server interface, but I think it's a lost cause. It just takes a slice of requests now.